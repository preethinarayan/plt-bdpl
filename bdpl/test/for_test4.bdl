int i;
int j;
i=0;
for(;;){
	for(j=0;j<5;j=j+1){
		if(j<i){
			print(j);
		}else{
			break;
		}
	}
	i=i+1;
	if(i<10) {
		print(i);
		continue ;
	}else{
		break;
	}
}