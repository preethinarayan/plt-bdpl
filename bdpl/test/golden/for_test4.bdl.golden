10201301240123501234601234701234801234901234